metadata/service/server/cluster.yml

```diff
@@ -28,6 +28,7 @@ parameters:
       role: ${_param:rabbitmq_cluster_role}
       master: ${_param:cluster_node01_hostname}
       mode: disc
+      partition_handling: autoheal
```

Comment:
Such defaults doesn't have to be in service-level metadata as they are not specific to deployment (single vs cluster). They belong to map.jinja or should default to `autoheal` where it gets rendered.